méli-mélo/méli-mélo-en.html

```diff
@@ -0,0 +1,95 @@
+---
+title: Méli-mélo features
+description: Reusable feature that are into a preliminary state of experimentation.
+lang: en
+altLangPage: méli-mélo-fr.html
+dateModified": 2021-05-27
+---
+
+<p>Reusable feature that are into a preliminary state of experimentation. Features are grouped into compilation in order to initiate quickly usability research, initiate preliminary discussion with key organisation and transforming the feature into a high quality produit adapted for GCWeb. Features are build from custom CSS/Javascript code.</p>
+
+<p>Every compilation life time are less than a year. That should provide feature sponsor department enough time to find resources to make their feature progressing as an official GCWeb feature. Support are provided during the wet-boew weekly code sprint happening remotely every Tuesday afternoon. An frozen (expired) méli-mélo compilation should not be used into any web page.</p>
+
+<p>This framework for méli-mélo compilation and feature are excluded from the GCWeb public API. Any change or removal would only trigger a patch release of GCWeb. That means the author are fully responsible but not required to documents any subsequent change they would make at their méli-mélo feature.</p>
+
+
+<h3>Méli-mélo sub-projects</h3>
+<p>(in their independent form)</p>
+
+{% assign melimelo_pages = site.pages | where: "output", "false" | where: "feature", "méli-mélo" | sort: "componentName" %}
+
+<!--
+Number of méli-mélo: {{ melimelo_pages | size }}
+-->
+
+<ul>
+{% for feature in melimelo_pages %}
+	{% assign first_example = feature.pages.examples | first %}
+	<li><strong>{{ feature.componentName }}</strong>: <a href="{{ feature.componentName }}/{{ first_example.path }}" lang="{{ first_example.language }}">{{ first_example.title }}</a></li>
+{% endfor %}
+</ul>
+
+<h2>Requirement</h2>
+
+<ul>
+	<li>Be reusable</li>
+	<li>Be sponsored by a department and by an active representative of that department</li>
+	<li>Must include an implementation plan that explain the steps taken by the sponsored department on how they will actively work the feature to get it integrated into GCWeb.</li>
+	<li>Must no contain any interference with wet-boew and GCWeb.</li>
+	<li>Each individual sub-feature and style must be demoed.</li>
+	<li>Working demo must be published using the GCWeb Jekyll template</li>
+	<li>Project name must be in this format. Year and month of the publication time of the feature followed by a short name identifying the feature</li>
+</ul>
+
+<h2>Contribution instruction</h2>
+
+<ul>
+	<li>Code or/and re-organize the feature by levering GCWeb Jekyll theme</li>
```

Comment:
It's always "and/or".  I've never seen "or/and" before.  "Reorganize" doesn't require a hyphen.